web/template/partial/stream/chat/chat_message.gohtml

```diff
@@ -38,18 +38,34 @@
                       x-text="watch.messageDateToString(m.CreatedAt)"></span>
             </div>
 
+            <!-- reactions -->
+            <div class="flex justify-between items-center">
+                <div class="flex items-center relative flex-wrap">
+                    <template x-for="(reactionGroup, i) in m.aggregatedReactions">
+                        <button type="button"
+                                :title="`${reactionGroup.names.join(', ')} reacted with :${reactionGroup.emojiName}:`" {{/* e.g. ':eyes:' */}}
```

Comment:
Personal preference but I would cap this off after a few names, e.g. "Khadim, Mathias and 7 others reacted with heart"